Makefile

```diff
@@ -33,8 +33,10 @@ CPPFLAGS = -D_XOPEN_SOURCE=700 \
 
 LDLIBS = -lImlib2 -lX11 -lXft -lfontconfig $(OPTIONAL_LIBS)
 
-OBJS = autoreload_$(AUTORELOAD).o commands.o image.o main.o options.o \
-  thumbs.o util.o window.o
+SRCS = src/autoreload_$(AUTORELOAD).c src/commands.c src/image.c src/main.c src/options.c \
+  src/thumbs.c src/util.c src/window.c
+
+OBJS = $(patsubst src/%.c, obj/%.o, $(SRCS))
```

Comment:
I'm not sure if this is POSIX, and we're trying to keep things POSIX compliant. Will need to look into this.